cmd/swapcli/main.go

```diff
@@ -184,15 +186,16 @@ func cliApp() *cli.App {
 					},
 					&cli.BoolFlag{
 						Name:  flagDetached,
-						Usage: "Exit immediately instead of subscribing to notifications about the swap's status",
+						Usage: "Exit immediately without subscribing to status notifications",
```

Comment:
Adding new aliases lengthens the help lines, so I scanned the help for places where I could shorten the lines.